stories/Tag/Tag.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import Tag from '../../components/Tag';
+
+const Template = args => {
+  const { children: Component, ...rest } = args;
+  return <Tag {...rest}>{Component}</Tag>;
+};
+
+export const Default = Template.bind({});
+
+export const Skin = Template.bind({});
```

Comment:
Are the "Default" and "Skin" examples exactly the same? Shouldn't there be some prop passed to differentiate one example from another?